tests/test_cli.py

```diff
@@ -32,13 +32,13 @@
 from mock import patch
 
 from invenio_search.cli import index as cmd
-from invenio_search.proxies import current_search, current_search_client
+from invenio_search.proxies import current_search_client
```

Comment:
Is this because of isort? Original version should be fine according to the rules, so hopefully isort doesn't say otherwise.

Reply:
It's was an unused import... I know we don't enforce removing these, so if you want I can leave it unchanged

Reply:
Ups :-) It's me who's blind hehe :)